src/export/InstanceExporter.ts

```diff
@@ -619,22 +643,12 @@ export class InstanceExporter implements Fishable {
       );
     }
 
-    // Since creating an instance of a Logical is not allowed,
-    // creating an instance of a Profile of a logical model is also not allowed
-    if (json.kind === 'logical' && json.derivation === 'constraint') {
-      throw new InstanceOfLogicalProfileError(
-        fshDefinition.name,
-        fshDefinition.instanceOf,
-        fshDefinition.sourceInfo
-      );
-    }
-
-    if (json.kind !== 'resource') {
+    if (json.kind !== 'resource' && json.kind !== 'logical') {
       // If the instance is not a resource, it should be inline, since it cannot exist as a standalone instance
       isResource = false;
       if (fshDefinition.usage !== 'Inline') {
         logger.warn(
-          `Instance ${fshDefinition.name} is not an instance of a resource, so it should only be used inline on other instances, and it will not be exported to a standalone file. Specify "Usage: #inline" to remove this warning.`,
+          `Instance ${fshDefinition.name} is not an instance of a resource or logical, so it should only be used inline on other instances, and it will not be exported to a standalone file. Specify "Usage: #inline" to remove this warning.`,
```

Comment:
We do use the keyword `Logical:` (not `LogicalModel:`) -- so referring to it as `logical` might not be a big stretch, but... I think you're right -- it would be better to say `logical model`.

Reply:
Fixed in 9e15356.